src/targeting.js

```diff
@@ -65,6 +66,13 @@ export const getHighestCpmBidsFromBidPool = hook('sync', function(bidsReceived,
   return bidsReceived;
 });
 
+/**
+ * Allow the configuration of TTL_BUFFER
+ */
+export function setTTLBuffer(buffer) {
```

Comment:
The [issue](https://github.com/prebid/Prebid.js/issues/8953) is about allowing the publisher to configure this, but this function is not accessible to the publisher. (The public publisher API consists only of what's exposed under the `pbjs` global).

The pattern we use elsewhere for this type of config is accepting them as parameters in `setConfig`. [Here's an example](https://github.com/prebid/Prebid.js/blob/0a14e91db750e3549bde19c371d1d34e877b6299/src/adapterManager.js#L50-L54) of registering a listener to watch for a specific config parameter. 

You'll need to choose a setting name (probably just `ttlBuffer`) which should be documented; and it'd be good to also have some test cases - `isBidNotExpired` should behave differently after different `setConfig`s.

Reply:
I missed that, thanks. I'll get on it.